src/travel-token-box/TravelTokenBox.tsx

```diff
@@ -141,7 +142,7 @@ const useStyles = (interactiveColor: InteractiveColor) =>
       marginBottom: theme.spacings.medium,
     },
     content: {
-      marginBottom: theme.spacings.large,
+      //marginBottom: theme.spacings.large, // re-add when new onboarding ready
```

Comment:
@marius-at-atb  This change was introduced for 1.46, now is merged into master this css style seems to be wrong now.

![image](https://github.com/AtB-AS/mittatb-app/assets/43450882/c18f7d83-aaec-40d8-bb11-7a38927a476d)

Space is missing for the bottom.

Reply:
Correct, everything in this PR was just for 1.46